.github/workflows/tests.yml

```diff
@@ -1,12 +1,20 @@
 name: Tests
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
```

Comment:
Could you explain the reason to change here?

Reply:
with that setup, I can run the action on my fork, to check if it is all fine before making a PR

see → https://github.com/LeFnord/ruby-oci8/actions

Reply:
I excluded this change from the commit https://github.com/kubo/ruby-oci8/commit/b9d83fcb707899ceaa16e8a7d26c233735a77d73 merged on the master branch.